support-frontend/assets/pages/showcase/showcase.jsx

```diff
@@ -57,7 +57,7 @@ const state = store.getState();
 
 const content = (
   <Provider store={store}>
-    <Page header={<CountrySwitcherHeader />} footer={<Footer />}>
+    <Page header={<CountrySwitcherHeader />} footer={<Footer faqsLink="" />}>
```

Comment:
What's the empty link here for?

Reply:
It _was_ to override the default FAQs link and not show one at all, but having reviewed the pages we're always overriding the default anywhere we need to show that link, so I'll have it behave the same as the Terms & Conditions link and not show unless it's passed.